backend/database/schema.sql

```diff
@@ -121,9 +121,8 @@ CREATE TABLE user (
     mail varchar(255)  NOT NULL,
     password varchar(255)  NOT NULL,
     week_time_kitchen int  NOT NULL,
-    sexe bool  NOT NULL,
     weight int  NOT NULL,
-    registration_date date  NOT NULL,
+    registration_date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Use registration_date DATETIME NOT NULL NOW() here

Reply:
Ok, it's good with that !